Flow.Launcher.Infrastructure/StringMatcher.cs

```diff
@@ -204,11 +187,16 @@ public MatchResult FuzzyMatch(string query, string stringToCompare, MatchOption
                 }
             }
 
-            // return acronym Match if possible
-            if (acronymMatchData.Count == query.Length && acronymScore >= (int) UserSettingSearchPrecision)
+            // return acronym match if all query char matched
+            if (acronymsMatched > 0 && acronymsMatched == query.Length)
             {
-                acronymMatchData = acronymMatchData.Select(x => translationMapping?.MapToOriginalIndex(x) ?? x).Distinct().ToList();
-                return new MatchResult(true, UserSettingSearchPrecision, acronymMatchData, acronymScore);
+                int acronymScore = (int)(acronymsMatched / acronymsTotalCount * 100);
+
+                if (acronymScore >= (int)UserSettingSearchPrecision)
+                {
+                    acronymMatchData = acronymMatchData.Select(x => translationMapping?.MapToOriginalIndex(x) ?? x).Distinct().ToList();
+                    return new MatchResult(true, UserSettingSearchPrecision, acronymMatchData, acronymScore);
+                }
             }
```

Comment:
Since we will check the whole string, do you think it will be better to compare acronym score and fuzzy search score and pick the higher one?

Reply:
hmm not sure, let me play with it, because it may get confusing which score it has returned, i would think it's better if acronym is used then that's the score it's returned, otherwise it adds new scenarios to the mix